### PR TITLE
Add partials for SDK dependency guidelines

### PR DESCRIPTION
**Pull request by paigecalvert**
- https://deploy-preview-1334--replicated-docs.netlify.app/vendor/helm-install-release
- https://deploy-preview-1334--replicated-docs.netlify.app/vendor/replicated-sdk-overview#how-to-distribute-the-sdk
- https://deploy-preview-1334--replicated-docs.netlify.app/vendor/replicated-onboarding#workflow

### CODE REVIEW

File: docs/vendor/replicated-sdk-overview.md [hunk 1]
```diff
@@ -13,17 +16,16 @@ This topic provides an introduction to using the Replicated SDK with your Helm c
...
-dependencies:
-- name: replicated-sdk
-  repository: oci://registry.replicated.com/library
-  version: 0.0.1-beta.1
-```
+<Dependency/>
+
+<SDKReleases/>
+
+Consider the following guidelines for adding the SDK as a dependency:
```
Comment (paigecalvert) [inline docs/vendor/replicated-sdk-overview.md:23]: This section was feeling a little hard to parse with all the sentences, so I tried out nesting them under a "guidelines" heading. WDYT?

File: docs/vendor/replicated-onboarding.md [hunk 2]
```diff
@@ -83,18 +87,14 @@ To begin onboarding to the Replicated platform with a Helm chart:
...
-   ```
+   <Dependency/>
 
    The Replicated SDK is a Helm chart that provides access to Replicated features and can be installed as a small service alongside your application. For more information, see [About the Replicated SDK (Beta)](/vendor/replicated-sdk-overview).
+   
+   <SDKReleases/>
 
    :::note
-   If your chart uses the Chart API version 1 (v1), then add the dependency to the `requirements.yaml` file rather than the `Chart.yaml` file. For more information, see [Consolidation of requirements.yaml into Chart.yaml](https://helm.sh/docs/faq/changes_since_helm2/#consolidation-of-requirementsyaml-into-chartyaml) in the Helm documentation.
+   <ChartRequirements/>
```
Comment (paigecalvert) [inline docs/vendor/replicated-onboarding.md:97]: I figured that, since there is a link about to the About section, we could get away with not including the note about "if you have multiple charts, put it in the chart that's installed first".

Also, this onboarding workflow is intentionally assuming that they are using a single Helm chart

File: docs/vendor/helm-install-release.md [hunk 3]
```diff
@@ -46,13 +48,11 @@ To package a Helm chart so that it can be added to a release:
...
-    dependencies:
-    - name: replicated-sdk
-      repository: oci://registry.replicated.com/library
-      version: 0.0.1-beta.1
-    ```
+    <Dependency/>
+
+    <SDKReleases/>
+    
+    For additional guidelines related to adding the SDK as a dependency, see [How to Distribute the SDK](replicated-sdk-overview#how-to-distribute-the-sdk) in _About the Replicated SDK_.
```
Comment (paigecalvert) [inline docs/vendor/helm-install-release.md:55]: Decided to just add a link here to the guidelines so as to not let the list get out of hand within the procedure

File: docs/vendor/replicated-sdk-overview.md (hunk 1 above)
Comment (jonquil2002) [reply docs/vendor/replicated-sdk-overview.md:20]: I like it, much easier to parse.

**Review by jonquil2002 — APPROVED** [review]
lgtm
